<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>singupwindow</class>
 <widget class="QDialog" name="singupwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;:/images/bg_login2.jpg&quot;);
background-position: center</string>
  </property>
  <widget class="QPushButton" name="btn_voltar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 82, 223);
font: 10pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QLabel" name="texto_frase">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0,79,0);
font: 9pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>&quot;Ler é viajar sem sair do lugar.&quot;
        - Mario Quintana</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>371</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="txt_cadastro">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 79, 0);
font: 20pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Faça seu cadastro!</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="txt_nome">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 79, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Nome</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="insert_nome">
        <property name="maxLength">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="txt_username">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 79, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="insert_username"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="txt_senha">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 79, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Senha</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="insert_senha">
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="txt_senha_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 79, 0);
font: 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Confirme sua senha</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="insert_senha_2">
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn_cadastro">
      <property name="styleSheet">
       <string notr="true">background: rgb(0, 79, 0);
color: rgb(255,255,255);
font: 9pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Cadastrar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="txt_erro">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);
font: 8pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>btn_voltar</zorder>
  <zorder>texto_frase</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
